src/inputs/radio/radio.jsx

```diff
@@ -231,7 +515,16 @@ class Radio extends React.Component {
                 />
 
                 {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-                <label className={labelClasses}>
+                <label
```

Comment:
Do we deliberately _not_ want to associate the label to an actual control element in this case?

Reply:
Yes, for now.

Reply:
Sounds good.  Resolving.